string file_name
string image_shape
---
bool is_right_image
